packages/expo-image/ios/ImageModule.swift

```diff
@@ -130,6 +130,22 @@ public final class ImageModule: Module {
       }
     }
 
+    AsyncFunction("getBlurhashAsync") {
+      (url: URL, numberOfComponents: CGSize, promise: Promise) in
```

Comment:
Besides the suggestion from SwiftLint, you can consider using `generateBlurhashAsync` instead of `getBlurhashAsync`. I guess it fits better to an action that might be time-consuming 🤔 
```suggestion
    AsyncFunction("generateBlurhashAsync") { (url: URL, numberOfComponents: CGSize, promise: Promise) in
```

Reply:
Good idea!